packages/frontend-2/lib/common/composables/toast.ts

```diff
@@ -68,7 +103,11 @@ export function useGlobalToast() {
    * Trigger a new toast notification
    */
   const triggerNotification = (notification: ToastNotification) => {
-    stateNotification.value = notification
+    const newNotification = { ...notification, id: uuidv4() }
```

Comment:
Another option would be to add the timeout to the `notification`, but that would mean both `triggerNotification` and `useGlobalToastManager` are concerned with timeout logic, open for suggestions